### ISSUE TITLE
Random dashboard order in a group

### ISSUE THREAD

**Issue by detouched** [signalfx/terraform-provider-signalfx#59]
When a group contains many dashboards, they appear in random order in SignalFX after first deploy. SignalFX keeps the order if I change it manually afterwards, but I wonder if this is something that can be enforced in the adapter.

**Comment by cory-signalfx** [comment]
Excellent question! I do not know the answer offhand and will have to dig internally to find out…

**Comment by cory-signalfx** [comment]
TL;DR: It's random and we likely can't do a lot about it.

I had to look at the implementation and ask a few questions. When the provider run does a group + dash run it creates a group so it has an ID, then creates all the child dashboards that are in that group. Terraform will parallelize when the items aren't dependent , so it's likely that these dashboards are created in whatever order Terraform feels / latency dictates. That pretty much means you're going to get random order.

There's been some discussion about using an alternate way of defining group membership, similarly to how we do charts today… but that would be a breaking change to the provider. :(
